api/KeyboardEvent.json

```diff
@@ -1175,7 +1175,7 @@
               "version_added": "56"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
Also false in IE, and I looked for `isComposing` in [Web Confluence](http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Edge_18.17763_Windows_10.0%22,%22Safari_14.0_OSX_10.15.6%22,%22Chrome_87.0.4280.66_Windows_10.0%22,%22Firefox_84.0_Windows_10.0%22%5D&q=%22isComposing%22), nothing suspicious found.